tests/e2e-playwright/page/LoginGoogle.js

```diff
@@ -0,0 +1,89 @@
+const { expect } = require("@playwright/test");
+const { selectors } = require('../utils/selectors')
+const {
+    WP_BASE_URL,
+    WP_USERNAME,
+    WP_PASSWORD
+} = require('../e2e-test-utils-playwright/src/config');
+
+exports.LoginGoogle = class LoginGoogle {
+    constructor(page) {
+        this.page = page;  
+    }
+
+// this function is used to navigate to Settings page
+async navigateToSettings(){
+    await this.page.goto("./wp-admin/options-general.php?page=login-with-google", { waitUntil: "load" });
+}
+// this function is used to validate setting's page Heading
+async validateHeading(){
+  await  expect(this.page.locator(selectors.settingsHeader)).toHaveText(selectors.settingsHeading);
+}
+// this functions is used to save settings
+async saveSetting() {
+ await this.page.locator(selectors.buttonSaveSetting).click();
+ expect(this.page.locator(selectors.messageSaveSetting)).not.toBeNull();
+}
+// this function is used to add proper secret and ID
+async setIdSecret(){
+    await this.page.click(selectors.clientIdInput, { clickCount: 3 })
+    await this.page.keyboard.press('Backspace')
+    await this.page.type(selectors.clientIdInput,selectors.clientIdValue)
+    await this.page.waitForTimeout(1000)
+    await this.page.click(selectors.clientSecretInput, { clickCount: 3 })
+    await this.page.keyboard.press('Backspace')
+    await this.page.type(selectors.clientSecretInput,selectors.clientSecretValue)
+}
+// this function is used to validate Login with google block is present in the editor
+async validateBlock(){
+    await this.page.type(selectors.titleInput,selectors.titleValue);
+    await this.page.click(selectors.addBlockButton);
+    await this.page.type(selectors.searchInput,selectors.searchInputValue);
+    expect(this.page.locator(selectors.validateOption)).not.toBe(null);
+}
+// this function is used to add Login with google block after validate Block
+async addLoginGoogleBlock(){
```

Comment:
@alvitazwar 
Add the assertions for this function. Validate whether the block is added successfully.